enhancements/installer/openstack-upi.md

```diff
@@ -0,0 +1,316 @@
+---
+title: openstack-upi
+authors:
+  - "@tomassedovic"
+reviewers:
+  - "@mandre"
+  - "@luis5tb"
+  - "@wking"
+  - "@cuppett"
+approvers:
+  - TBD
```

Comment:
can we update the approvers list?

I think it should be at least @sdodson and @crawford